README.md

```diff
@@ -128,51 +130,75 @@ involve IPFS.
 We will be using the following tools and services:
 
 * [Digital Ocean](https://www.digitalocean.com) as the virtual machine provider
-* [Vagrant](https://www.vagrantup.com) to provision the cloud servers
+* [Terraform](https://www.terraform.io) to provision the cloud servers
 
 The following steps assume you have a Digital Ocean account and the above listed software
 installed on your local machine, which can be the same device running OBS Studio.
 
-1. Install vagrant plugins:
+1. Clone this repository and work from the `terraform` directory:
+
+        git clone https://github.com/tomeshnet/ipfs-live-streaming.git
+        cd ipfs-live-streaming/terraform
+
+1. From your domain name registerar, point name servers to Digital Ocean's name servers:
 
-        vagrant plugin install vagrant-digitalocean vagrant-scp
+        ns1.digitalocean.com
+        ns2.digitalocean.com
+        ns3.digitalocean.com
+
+    Then store the domain name in your local environment:
+
+        echo -n YOUR_DOMAIN_NAME > .keys/domain_name
```

Comment:
.key folder does not exists at this point in time.

Reply:
It should exist because of the clone, and the `.keys` folder has a `.gitkeep`

Reply:
Oh nevermind I `.gitignored` it :(